cluster/addons/fluentd-elasticsearch/fluentd-es-ds.yaml

```diff
@@ -23,28 +70,14 @@ spec:
     spec:
       serviceAccountName: fluentd-es
       containers:
-      - name: fluentd-es
-        image: gcr.io/google_containers/fluentd-elasticsearch:1.24
-        command:
-          - '/bin/sh'
-          - '-c'
-          - '/usr/sbin/td-agent $FLUENTD_ARGS'
+      - name: logging-agent
+        image: gcr.io/google-containers/fluentd-elasticsearch:v0.2.0
         env:
         - name: FLUENTD_ARGS
-          value: -q
-        - name: FLUENT_ELASTICSEARCH_USER
-          valueFrom:
-            configMapKeyRef:
-              name: environment
-              key: elasticsearch-user
-        - name: FLUENT_ELASTICSEARCH_PASSWORD
-          valueFrom:
-            secretKeyRef:
-              name: environment
-              key: elasticsearch-password
+          value: --no-supervisor -q
         resources:
           limits:
-            memory: 200Mi
+            memory: 500Mi
```

Comment:
Empirically. Note that it's limit, I haven't changed the request. that's just to protect the fluentd from ruby having memory spikes